src/App.tsx

```diff
@@ -10,9 +10,19 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="people">
+              <Route index element={<PeoplePage />} />
+              <Route path=":personSlug" element={<PeoplePage />} />
+            </Route>
+
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
```

Comment:
Create separate components for this case
```suggestion
            <Route path="/" element={...} />
            <Route path="home" element={<Navigate to="/" replace />} />
            <Route
              path="*"
              element={...}
```

Reply:
Do it)